api/swagger/swagger-v1.yaml

```diff
@@ -3279,6 +3379,104 @@ paths:
         "500":
           description: Server error
           content: {}
+  /coins:
+    get:
+      tags:
+        - coins
+      operationId: Get Coins
+      description: 'Gets information about coins'
+      parameters:
+      - name: mint
+        in: query
+        description: The mint addresses of the coins
+        style: form
+        explode: true
+        schema:
+          type: array
+          items:
+            type: string
+      - name: owner_id
+        in: query
+        description: The user IDs of artists with minted coins
+        style: form
+        explode: true
+        schema:
+          type: array
+          items:
+            type: string
+          
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/coin'
+  /coins/{mint}:
+    get:
+      tags:
+        - coins
+      operationId: Get Coin
+      description: 'Gets information about a specific coin by its mint address'
+      parameters:
+      - name: mint
+        in: path
+        description: The mint address of the coin
+        required: true
+        schema:
+          type: string
+          example: 9LzCMqDgTKYz9Drzqnpgee3SGa89up3a247ypMj2xrqM
+      responses:
+        '200':
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/coin'
+  /coins/{mint}/members:
+    get:
+      tags:
+        - coins
+      operationId: Get Coin Members
+      description: 'Gets a list of Audius users with a non-zero balance of a specific coin'
+      parameters:
+      - name: mint
+        in: path
+        description: The mint address of the coin
+        required: true
+        schema:
```

Comment:
I assume we will want pagination for this endpoint? And for all of the endpoints that return potentially large amounts of things (like `/coins`)